web/src/app/theme/themeConfig.tsx

```diff
@@ -26,22 +25,41 @@ export const ThemeContext = React.createContext<ThemeContextParams>({
 })
 ThemeContext.displayName = 'ThemeContext'
 
-function getPalette(mode: ThemeName): PaletteOptions {
-  if (mode === 'dark') {
+// palette generated from https://material-foundation.github.io/material-theme-builder/#/custom
+function getPalette(mode: string): PaletteOptions {
+  const prefersDark = window.matchMedia('(prefers-color-scheme: dark)')
+
+  if (mode === 'light' || (mode === 'system' && !prefersDark)) {
     return {
-      mode: 'dark',
-      secondary: grey,
+      mode: 'light',
+      primary: {
+        main: '#006684',
+        light: '#bbe9ff',
+        dark: '#001f2a',
+      },
+      secondary: { main: '#4d616b', light: '#d0e6f3', dark: '#081e27' },
+      background: {
+        default: '#fbfcfe',
+        paper: '#dce3e8', // m3 surface variant
+      },
+      error: { main: '#ba1b1b', light: '#ffdad4', dark: '#410001' },
     }
   }
 
-  return {
-    mode: 'light',
-    primary: {
-      ...grey,
-      main: '#616161',
-    },
-    secondary: grey,
+  if (mode === 'dark' || (mode === 'system' && prefersDark)) {
+    return {
+      mode: 'dark',
+      primary: { main: '#64d3ff', light: '#bbe9ff', dark: '#004d65' },
+      secondary: { main: '#b5cad6', light: '#d0e6f3', dark: '#354a54' },
+      background: {
+        default: '#191c1e',
+        paper: '#191c1e', // m3 surface
+      },
+      error: { main: '#ffb4a9', light: '#ffdad4', dark: '#930006' },
+    }
   }
+
+  return {}
```

Comment:
I think we should fall back to light, instead of adding a 3rd mode of "whatever MUI defaults are". Also looks like maybe merge conflicts lost the type info

ThemeName can only be `'light'` or `'dark'` we don't need to handle `'system'` here anymore

Reply:
System is one of the options to store, has that changed recently?